lib/generators/module/USAGE

```diff
@@ -0,0 +1,8 @@
+Description:
+    Explain the generator
```

Comment:
Should you fill out this file with basically the content in this PR's description?